File: leetcode/leetcode.js
const readline = require('readline');

let rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout 
})

function item1(){
    console.log("this is busted");
}


function main(){
    console.log("\nSelect which item do you want to print from below list:\n");
    console.log("1. Bitwise operations");

    rl.question("", (num_of_items) => { 
        num_of_items = parseInt(num_of_items);
        console.log();

        switch (num_of_items) {
            case 1:
                item1();
                break;
        
            default:
                console.log("\nInvalid Number, Please select a number from above given list :)\n");
                rl.close();
                break;
        }
    } )
}

main();

File: leetcode/leetcode.py

def item1():
    print("This is console")

def invalid_item():
    print("Invalid value")

def main():
    print("\nSelect which item do you want to print from below list:)\n")
    print("\n1. Bitwise opeations\n")

    num_of_numbers = int(input())
    print()


    items = {
        1: item1,
    }

    items.get(num_of_numbers, invalid_item)()

if __name__ == "__main__": 
    main()
